content/cds_hooks.md

```diff
@@ -0,0 +1,117 @@
+---
+title: CDS Hooks
+layout: cds_hooks
+---
+
+# CDS Hooks #
+------------------------------------------------------------------------
+<%= beta_tag(resource: false) %>
+* TOC
+{:toc}
+
+## General ##
+
+### Introduction ###
```

Comment:
Should we have a note at the end of the introduction to mention CDS Hooks is dependent on specific software releases and then point to our reference pages? I don't want to put the actual releases on our reference page since we've had to change them frequently. I know 3rd party developers won't have access to those reference pages but clients will. Thoughts?